test/mpi/rma/get_acc_flush.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <unistd.h>
+#include <mpi.h>
+#include "mpitest.h"
+
+#define GET_BUF_SIZE 2147467264LL
+#define GET_PER_SIZE 16384
+#define ACC_BUF_SIZE 32768
+#define SIGNATURE 0x77777777
+#define ITERATION 10
+
+/* This program tests the rma issue counter bug located at MPIDI_OFI_win_do_progress function.
+ * It needs at least 4 processes launched in round-robin way on two nodes; for example, P0 and P2
+ * are on node0, and P1 and P3 should be on node1.
+ */
+
```

Comment:
I don't understand why it has to be round-robin. Besides, can you add short description of the test program? E.g., P0 issues large get to P1 while other ranks issues accumulate to P0.

Reply:
One more comment for the header comment: "the rma issue counter bug located at MPIDI_OFI_win_do_progress function." is not a complete description of the situation. 

I guess it is better to say "this program tries to mimic the case where rdma-get are issued both for RMA get and rdma-get-based AM on a process (i.e., P0 in this program). The netmod internal counters may not be accurately updated at window flush, thus flush may return before the completion of all RMA get operations."

Reply:
Round robin is not necessary. The current program assumes rank 0 and 1 are on different nodes. I can change it to make it more general.

Reply:
Can you please make "netmod internal counters update bug during win flush" more specific? For example, you can say "may trigger OFI internal out-of-order RDMA cntr update (e.g., AM-based acc updates cntr before last RMA get on rank 0) and thus completes win flush incorrectly".

Besides, why not simply mention P0 or P1 instead of process i, j?